### PR TITLE
[FAB] Restore iOS to prior state / Fix Android elevation

### PR DESCRIPTION
**Pull request by ayush547**
### Platforms Impacted
- [x] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [x] android

### Description of changes
Undoing changes made in PR [#2366](https://github.com/microsoft/fluentui-react-native/pull/2366)
Reverting FAB on iOS to prior appearance.
Added FAB to Button Variants Section, removed Small FAB from Button Size Section for iOS.

Minor - Fix elevation not being applied on FAB Android

### Verification

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| image |  image |

### Pull request checklist

This PR has considered (when applicable):
- [x] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/fluent-tester/src/TestComponents/Button/ButtonVariantTestSection.tsx [hunk 1]
```diff
@@ -39,6 +40,9 @@ export const ButtonVariantTest: React.FunctionComponent = () => {
       <CompoundButton appearance="subtle" secondaryContent="Compound" style={commonTestStyles.vmargin}>
         Subtle
       </CompoundButton>
+      <FAB icon={iconProps} style={commonTestStyles.vmargin}>
```
Comment (lyzhan7) [inline apps/fluent-tester/src/TestComponents/Button/ButtonVariantTestSection.tsx:43]: Seems like for Android we're already showing this in the ButtonSizeTestSection. Do we also want to show it for Android here, or should this be just shown on iOS?

File: packages/components/Button/src/FAB/FABTokens.ts [hunk 2]
```diff
@@ -75,4 +75,5 @@ export const defaultFABTokens: TokenSettings<FABTokens, Theme> = (t: Theme) =>
         spacingIconContentBefore: globalTokens.size80,
       },
     },
+    shadowToken: t.shadows.shadow8,
```
Comment (lyzhan7) [inline packages/components/Button/src/FAB/FABTokens.ts:78]: I don't think this is a necessary for either iOS or android shadows - iOS will be referencing the FABTokens.ios.ts file instead, and the shadowToken shouldn't have any effect on Android shadows

**Comment by lyzhan7** [comment]
Just pulled your PR to test, for reference here's a video of the tester:

Uploading Screen Recording 2022-12-02 at 11.18.20 AM.mov…

Overall looks good, had a couple notes:
- just wanted to confirm that the new FABTokens.ios.ts and FABColorTokens.ios.ts files were copied from what the old FABTokens.ts and FABColorTokens.ts files were before the Android changes?
- the spacing between the icon and the text in the before/after screenshots seems to have changed, do you know what happened here?

File: apps/fluent-tester/src/TestComponents/Button/ButtonVariantTestSection.tsx (hunk 1 above)
Comment (ayush547) [reply apps/fluent-tester/src/TestComponents/Button/ButtonVariantTestSection.tsx:43]: For Android it is displayed in both Button Variant section (forked out) and the Button sizes section. On iOS it was initially being shown, have brought back the old behaviour

File: packages/components/Button/src/FAB/FABTokens.ts (hunk 2 above)
Comment (ayush547) [reply packages/components/Button/src/FAB/FABTokens.ts:78]: Agreed, removing it

**Comment by ayush547** [comment]
> Just pulled your PR to test, for reference here's a video of the tester:
> 
> Uploading Screen Recording 2022-12-02 at 11.18.20 AM.mov…
> 
> Overall looks good, had a couple notes:
> 
> * just wanted to confirm that the new FABTokens.ios.ts and FABColorTokens.ios.ts files were copied from what the old FABTokens.ts and FABColorTokens.ts files were before the Android changes?
> * the spacing between the icon and the text in the before/after screenshots seems to have changed, do you know what happened here?

1. Yes they were picked from the previous one, for reference, [FABColorTokens](https://github.com/ayush547/fluentui-react-native/blob/890ff26064088ee92d44bb8730b8629143533451/packages/components/Button/src/FAB/FABColorTokens.ts) & [FABTokens](https://github.com/ayush547/fluentui-react-native/blob/890ff26064088ee92d44bb8730b8629143533451/packages/components/Button/src/FAB/FABTokens.ts)
2. Thank you for catching that, have fixed it now

**Comment by lyzhan7** [comment]
Changes to the iOS FAB spacing look good, thanks for fixing that

Screenshot 2022-12-02 at 3 48 22 PM

**Review by lyzhan7 — APPROVED** [review]

**Pull request merged**